Article "Spécifier l'accessibilité de vos designs grâce aux annotations" en FR et  EN (#781)
Orange-OpenSource/a11y-guidelines: src/en/articles/accessibility-annotations.md

---
title: "Specify the accessibility of your designs by using annotations"
abstract: "Accessibility annotations allow you to specify how the different components present on your wireframe should be interpreted by assistance tools"
titleBeforeTag: true
date: "2025-11-04"
updateDate : "2025-11-04"
tags:
  - beginner
---

## What is it about ?

Accessibility annotations specify how the various components on a screen should be interpreted by assistive tools (like a screen reader) and help to anticipate several accessibility error risks. while some annotations can be added directly by designers during the graphic design phase, others require technical expertise.
This is the main benefit of this activity, which encourages a cross-disciplinary reflection on accessibility (designer, product manager, accessibility expert, developer, etc.).

> Without these specifications, technical accessibility rest solely on the shoulders of developers, so it must be above all a team effort.

Reviewer: ```suggestion
Without these specifications, technical accessibility rests solely on the shoulders of developers, while it must be, above all, a team effort.
```

## What should be annotated ?

Some informations that can be added to your designs&nbsp;: 
- Text alternative for images
- Title of each page
- Headings (HTML tags `h1`, `h2`...)

> - Semantics (lists, paragraphs, quotations, etc.)

Reviewer: ```suggestion
- Semantics (lists, paragraphs, quotations...)
```

- list of the different page regions (landmarks)
- Interactive components (links, buttons...)
- Reading order of components with screen reader
- Focus moving order
- Forms (labels, mandatory fields, error messages...)
- Keyboard shortcuts (for example Esc to close a dialog box)
- Skip links...

<img src="../images/annotations/example-en.png" alt="Design example with accessibility annotations" title="Design example with accessibility annotations" class="img-fluid">

> ## How to do it ?

Reviewer 2: ```suggestion
## How to do it?
```

> Whatever software used, annotating mockups is really easy. For example, you can use numbered bullet points that link to a detailed description. There are also annotation kits that offer stickers with different colors and shapes that allow you to quickly annotate your designs. The best tool is the one that best suits your way of working. It may vary depending on the profiles and size of your team. Annotation kits provide a foundation that you should feel free to adapt to your needs.

Reviewer: ```suggestion
Whatever software used, annotating mockups is really easy. For example, you can use numbered bullet points that link to a detailed description. There are also annotation kits or plugins that offer stickers with different colors and shapes that allow you to quickly annotate your designs. The best tool is the one that best suits your way of working. It may vary depending on the profiles and size of your team. Annotation kits, as plugins, provide a foundation that you should feel free to adapt to your needs.
```

Some examples of information that can be added to your designs:&nbsp;: 
- <a href="https://stephaniewalter.design/blog/a-designers-guide-to-documenting-accessibility-user-interactions/" lang="en" hreflang="en">A Designer’s Guide to Documenting Accessibility & User Interactions</a>
- <a href="https://tangible.is/en/thinking/2024/10/the-value-of-accessibility-annotations-in-inclusive-design" lang="en" hreflang="en">The value of accessibility annotations in inclusive design</a>
- <a lang="en" hreflang="en" href="https://medium.com/indeed-design/building-an-accessibility-library-e134e9012c17">Building an Accessibility Library</a>
- <a hreflang="en" href="https://blog.zeplin.io/introducing-flows-and-annotations-a-new-way-to-communicate-design-intention">Introducing Flows and Annotations</a>
- <a lang="en" hreflang="en" href="https://www.youtube.com/watch?v=Y35jmpS8lQM&feature=youtu.be&ab_channel=InclusiveDesign24%23id24">Annotating designs for Accessibility / Claire Webber and Sarah Pulis</a>
- <a lang="en" hreflang="en" href="https://tetralogical.com/blog/2025/09/23/annotating-designs-using-common-language/">Annotating designs using common language / Craig Abbott</a>

> Some Figma plugins to add accessibility annotations&nbsp;: 

Reviewer: ```suggestion
Some Figma plugins to add accessibility annotations: 
```
